cmd/minikube/cmd/start.go

```diff
@@ -357,6 +358,22 @@ func validateConfig() {
 			exit.Usage("Sorry, the kubeadm.%s parameter is currently not supported by --extra-config", param)
 		}
 	}
+
+	// This validates if the --registry-mirror args
+	// match the format of http://localhost
+	if len(registryMirror) > 0 {
```

Comment:
Thanks for the changes to use ur.Parse. I suggest moving this logic to a function and also add the unit tests for that function. example can be found here https://github.com/kubernetes/minikube/blob/master/cmd/minikube/cmd/env_test.go#L71